chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -528,21 +534,32 @@ impl PeerManagerActor {
             self.routing_table_addr
                 .send(RoutingTableMessages::AddVerifiedEdges { edges: new_edges })
                 .into_actor(self)
-                .map(move |response, act, _ctx| match response {
-                    Ok(RoutingTableMessagesResponse::AddVerifiedEdgesResponse(filtered_edges)) => {
-                        // Broadcast new edges to all other peers.
-                        if act.adv_helper.can_broadcast_edges() {
-                            let sync_routing_table = RoutingTableUpdate::from_edges(filtered_edges);
-                            Self::broadcast_message(
-                                network_metrics,
-                                &act.connected_peers,
-                                SendMessage {
-                                    message: PeerMessage::SyncRoutingTable(sync_routing_table),
-                                },
-                            )
+                .map(move |response, act, _ctx| {
+                    let _span = tracing::trace_span!(
+                        target: "network",
+                        parent: &span,
```

Comment:
This one I believe is buggy, see https://near.zulipchat.com/#narrow/stream/328640-temp-stakewars-prep/topic/shardnet.20debug for details.